src/test/java/ru/training/at/hw2/ex1/Exercise1Tests.java

```diff
@@ -0,0 +1,147 @@
+        WebElement contactFormMenuHead =
+            driver.findElement(By.cssSelector("[class=\"uui-header dark-gray\"] a[href=\"contacts.html\"]"));
+        assertEquals(contactFormMenuHead.getText(), "CONTACT FORM");
+        //        Service
+        WebElement servicesMenuHead =
+            driver.findElement(By.cssSelector("[class=\"uui-header dark-gray\"] a[class=\"dropdown-toggle\"]"));
+        assertEquals(servicesMenuHead.getText(), "SERVICE");
+        //        Metals & Colors
+        WebElement metalsAndColorsMenuHead =
+            driver.findElement(By.cssSelector("[class=\"uui-header dark-gray\"] a[href=\"metals-colors.html\"]"));
+        assertEquals(metalsAndColorsMenuHead.getText(), "METALS & COLORS");
+
+        //        6. Assert that there are 4 images on the Index Page and they are displayed
+        //        Image 1
+        driver.findElement(
+            By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(1) div[class=\"benefit-icon\"]"));
+        //        Image 2
+        driver.findElement(
+            By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(2) div[class=\"benefit-icon\"]"));
+        //        Image 3
+        driver.findElement(
+            By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(3) div[class=\"benefit-icon\"]"));
+        //        Image 4
+        driver.findElement(
+            By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(4) div[class=\"benefit-icon\"]"));
+
+        //        7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        //        Text 1
+        WebElement textUnderMicroscope =
+            driver.findElement(
+                By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(1) span[class=\"benefit-txt\"]"));
+        assertEquals(textUnderMicroscope.getText(), "To include good practices\nand ideas from successful\n"
+            + "EPAM project");
+        //        Text 2
+        WebElement textUnderHeadphones =
+            driver.findElement(By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(2) "
+                + "span[class=\"benefit-txt\"]"));
+        assertEquals(textUnderHeadphones.getText(), "To be flexible and\ncustomizable");
+        //        Text 3
+        WebElement textUnderMonitor =
+            driver.findElement(
+                By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(3) span[class=\"benefit-txt\"]"));
+        assertEquals(textUnderMonitor.getText(), "To be multiplatform");
+        //        Text 4
+        WebElement textUnderRocket = driver.findElement(
+            By.cssSelector("div[class=\"row clerafix benefits\"] > div:nth-child(4) span[class=\"benefit-txt\"]"));
+        assertEquals(textUnderRocket.getText(), "Already have good base\n(about 20 internal and\n"
+            + "some external projects),\nwish to get more…");
```

Comment:
There are more suitable locators. You shouldn't use indexes in locators if you can. In this situation, you can find locators without indexes.  Maybe you can take a look on xpath?